### PR TITLE
FE-92: Display support docs related to current page in app

### PR DESCRIPTION
**Pull request by bkemper**

### CODE REVIEW

**Comment by bkemper** [comment]
@ewandennis I just pushed a fix for the tests failing, so this PR is ready for final review.

File: src/components/index.js [hunk 1]
```diff
@@ -16,7 +16,6 @@ export { default as PieChart } from './pieChart/PieChart';
 export { default as PlanPicker } from './planPicker/PlanPicker';
 export { default as SparkPost } from './sparkPost/SparkPost';
 export { default as CenteredLogo } from './sparkPost/CenteredLogo';
-export { default as Support } from './support/Support';
```
Comment (ewandennis) [inline src/components/index.js]: This is smart.

File: src/components/support/Support.js [hunk 2]
```diff
@@ -48,7 +50,8 @@ export class Support extends Component {
   }
 
   render() {
-    const { loggedIn, showPanel, showTicketForm } = this.props;
+    const { location, loggedIn, showPanel, showTicketForm } = this.props;
+    const keyword = _.get(findRouteByPath(location.pathname), 'keywords[0]');
```
Comment (ewandennis) [inline src/components/support/Support.js]: Don't think we need a list of keywords in our config if we're just going to pick out the first one. Also, Algolia will search on a _phrase_ so maybe we should just use the term "query".

File: src/components/support/components/AlgoliaSearch.js [hunk 3]
```diff
@@ -1,14 +1,50 @@
...
+import React, { Component } from 'react';
 import { TextField } from '@sparkpost/matchbox';
 import { connectSearchBox } from 'react-instantsearch/connectors';
 
-export const AlgoliaSearch = ({ currentRefinement, refine }) =>
-  <TextField
-    onChange={(e) => refine(e.currentTarget.value)}
-    value={currentRefinement}
-    placeholder={'Have a question?  Ask or enter a search term here.'}
+// TODO: https://github.com/algolia/react-instantsearch/issues/355
```
Comment (ewandennis) [inline src/components/support/components/AlgoliaSearch.js]: Slightly glib TODO message here. If Algolia's duplicate calls is affecting our UX, lets cut an improvement / defect and remove this.

File: src/components/support/components/tests/AlgoliaSearch.test.js [hunk 4]
```diff
@@ -4,23 +4,46 @@ import { shallow } from 'enzyme';
...
 
-  it('should render', () => {
+  it('should render with empty text field', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
-  it('should call refine on change', () => {
-    wrapper.find('TextField').simulate('change', { currentTarget: { value: 'foo' }});
-    expect(props.refine).toHaveBeenCalledWith('foo');
+  it('should refine search on mount wiht default search text', () => {
```
Comment (ewandennis) [inline src/components/support/components/tests/AlgoliaSearch.test.js]: `s/wiht/with/`

File: src/config/routes.js [hunk 5]
```diff
@@ -169,10 +169,13 @@ const routes = [
     condition: all(
       hasGrants('api_keys/manage', 'templates/modify', 'sending_domains/manage'),
       configEquals('splashPage', '/dashboard') // want to hide if not a splash page https://jira.int.messagesystems.com/browse/FAD-6046
-    )
+    ),
     // TODO: implement some kind of blockedRoutes check that runs on every route so we can
     // hide routes based on config, account/user settings, etc. without having to mess
     // around with grants in the web UI keys
+    keywords: [
```
Comment (ewandennis) [inline src/config/routes.js]: As I mentioned above, lets just keep a query string per route.

File: src/components/support/Support.js (hunk 2 above)
Comment (bkemper) [reply src/components/support/Support.js]: Agreed.  This was a premature optimization.  I'll revert it.

**Comment by bkemper** [comment]
I am going to wait to address feedback after #490 is merged.

File: src/components/support/Support.js (hunk 2 above)
Comment (bkemper) [reply src/components/support/Support.js]: Renamed to `supportDocSearch`.

**Pull request merged**